pkg/webhook/admission/builder/doc.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package builder provides methods to build admission webhooks.
+
+For most users, we recommend to use Operations and ForType to construct a webhook,
+since it is more intuitive and easier to pass the target operations to Operations method and
+a empty target object to ForType method than passing a complex RuleWithOperations struct to Rules method.
```

Comment:
Maybe move this explanation to the later part of the doc and show the example webhooks first.
```
since it is more intuitive and easier to pass the target operations to Operations method and
a empty target object to ForType method than passing a complex RuleWithOperations struct to Rules method.
```